@kenglxn @shawnlauzon I'm getting a compile error with this PR, which I would think is related to the `AndroidSetup_r17` class.  I haven't done a comparison on what exactly changed, but it compiles fine in master.  Exact error is:

```
21:13:35.635 [ERROR] [org.gradle.BuildExceptionReporter] Caused by: : Forked groovyc returned error code: 1
21:13:35.637 [ERROR] [org.gradle.BuildExceptionReporter]    at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
21:13:35.639 [ERROR] [org.gradle.BuildExceptionReporter]    at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
21:13:35.640 [ERROR] [org.gradle.BuildExceptionReporter]    at org.gradle.api.internal.project.ant.BasicAntBuilder.nodeCompleted(BasicAntBuilder.java:71)
21:13:35.640 [ERROR] [org.gradle.BuildExceptionReporter]    at org.gradle.api.internal.project.AntBuilderDelegate.nodeCompleted(DefaultIsolatedAntBuilder.groovy:173)
21:13:35.641 [ERROR] [org.gradle.BuildExceptionReporter]    at org.gradle.api.internal.tasks.compile.AntGroovyCompiler$_execute_closure1.doCall(AntGroovyCompiler.groovy:60)
```

Are you guys encountering the same thing?

--- a/src/main/groovy/com/jvoegele/gradle/plugins/android/AndroidSetupFactory.groovy
+++ b/src/main/groovy/com/jvoegele/gradle/plugins/android/AndroidSetupFactory.groovy
@@ -42,12 +42,12 @@ class AndroidSetupFactory {
   }
 
   AndroidSetup getAndroidSetup() {
-	if (this.androidSdkToolsRevision < 14) {
-	  return new AndroidSetup_r13(project)
-	} else if (this.androidSdkToolsRevision >= 17) {
-    return new AndroidSetup_r17(project)
-  } else {
-	  return new AndroidSetup_r14(project)
-	}
+    if (this.androidSdkToolsRevision < 14) {
+      return new AndroidSetup_r13(project)
+    } else if (this.androidSdkToolsRevision < 17) {
+      return new AndroidSetup_r14(project)
+    } else {
+      return new AndroidSetup_r17(project)
+    }
   } 
 }

